File: src/test/programmers/java/level1/MakeDecimal.java
package test.programmers.java.level1;

public class MakeDecimal {
    public static void main(String[] args) {
        int[] arr = {1, 2, 7, 6, 4};

        System.out.println(solution(arr));
    }

    static int solution(int[] nums) {
        int answer = 0;

        for(int i = 0; i < nums.length - 2; i++) {
            for(int j = i + 1; j < nums.length - 1; j++) {
                for(int k = j + 1; k < nums.length; k++) {
                    int value = nums[i] + nums[j] + nums[k];
                    if(chkDecimal(value)) {
                        answer += 1;
                    }
                }
            }
        }

        return answer;
    }

    static boolean chkDecimal(int value) {
        int count = 0;

        for(int i = 1; i <= value; i++) {
            if (value % i == 0) count++;
        }

        return count <= 2 ? true : false;
    }
}

File: src/test/programmers/python/level1/MakeDecimal.py
def solution(nums):
    answer = 0
    for i in range(len(nums) - 2):
        for j in range(i + 1, len(nums) - 1):
            for k in range(j + 1, len(nums)):
                num = nums[i] + nums[j] + nums[k]
                n = 0
                for p in range(1, num + 1):
                    if num % p == 0:
                        n += 1
                if n == 2:
                    answer += 1
    return answer


# print(solution([1,2,7,6,4]))